getting_started/build-guide/build-from-scratch.rst

```diff
@@ -30,13 +30,16 @@ modifications.
    because some default configuration files refer to other clones using
    relative locations like ``../sof/``.
 
+   Make sure that ``$SOF_WORKSPACE`` have adequate amount of disk space. 100GB
+   or more is recommended.
```

Comment:
Please say most of it is for the toolchains because many people don't bother creating the toolchains and use the docker image instead.

Reply:
So this is only if we build the toolchain, we could have that as an option, probably making the download as the default .

Reply:
> Please say most of it is for the toolchains because many people don't bother creating the toolchains and use the docker image instead.

@marc-hb, I'll update to be specific that the space is needed for building the toolchain. This is the build from scratch guide, not the build with docker.

Reply:
@ujfalusi I wonder if we can make the toolchains available somewhere so the user only needs to download them.

Reply:
> This is the build from scratch guide, not the build with docker.

Sorry I forgot that "build from scratch" means "not with Docker". About half of the "build from scratch" content applies to Docker the same, now I realize that there has been some serious copy/paste/diverge between the two. I suggest maintaining that "common" half like it would actually become common in the future.

>  I wonder if we can make the toolchains available somewhere so the user only needs to download them.

That would be nice. This by far the most disk, CPU and time-consuming part of this section.

Reply:
@dbaluta, I don't know if we have the infra for file server, but it would be a good thing.
Personally I feel safer to build it for myself, it has to be done only once and I know that it will work on my non Ubuntu Linux ;)

@marc-hb, @lrgirdwo, do you have suggestions on storing the toolchains?

Reply:
> @marc-hb, @lrgirdwo, do you have suggestions on storing the toolchains?

The most natural place would be either at https://github.com/thesofproject/xtensa-overlay/releases or at https://github.com/thesofproject/crosstool-ng/releases and - surprise - there is already something in both places.

Reply:
@marc-hb, great. I can send an update later to document the option to download and not build the toolchain form the releases of both xtensa and crosstool-ng.
Most likely later this week or next week?

Reply:
@marc-hb cool!!